### ISSUE TITLE
[Bug] enableStrictSsl config option is NOT available in the latest release

### ISSUE THREAD

**Issue by dkelly99** [yarnpkg/berry#2169]
- [ ] I'd be willing to implement a fix

**Describe the bug**

The config option `enableStrictSsl` (described [here](https://yarnpkg.com/configuration/yarnrc#enableStrictSsl)) is **NOT** available in the latest release.  I need to set to `false` but i just get an error.
I followed a previous issue #798 which details how the config was introduced in PR #1934 .  That PR was merged but the new config option is not in master.  

**Error Message**
`Usage Error: Couldn't find a configuration settings named "enableStrictSsl"`

**To Reproduce**

<details>
  <summary>Reproduction</summary>

  1. `yarn set version berry` or `yarn set version latest`
  2. `yarn config set enableStrictSsl false`
</details>

**Screenshots**
![image](https://user-images.githubusercontent.com/4181845/100294331-6281f180-2f7e-11eb-8f79-b13c1ce0e391.png)

**Comment by arcanis** [comment]
That's expected - the website uses our master branch, so it's unfortunately slightly ahead of itself. You can try out the new changes with [`yarn set version from sources`](https://yarnpkg.com/cli/set/version/from/sources), or wait for the next release to be shipped (end of week).

**Comment by dkelly99** [comment]
OK, thanks... I saw that PR #1934 had been merged to master so I assumed that it was in latest version.  Perhaps the documentation should align with the available functionality or even a note in the docs to say that the functionality is not yet in the latest release?  

I had already tried `yarn set version from sources` but that failed... it just hangs.  In the meantime I managed to grab the artifact from PR #1934 and inserted it into my project, which worked perfectly.  I will replace that when you guys release the latest version.  BTW, I love how the yarn version is now packed inside the project itself.   It makes it easier to work around this kind of issue.  Thanks again.
